### PR TITLE
Added documentation for Bitvavo integration

### PR DESCRIPTION
**Pull request by cyberjunky**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48042
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2342
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bitvavo.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+With the Bitvavo integration, you can keep track of cryptocurrencies from the [Bitvavo](https://bitvavo.com) exchange.
+
+## Requirements
+
+Setup requires an API Key and Secret Key from Bitvavo with at least 'Read' permissions. Browse to [API Settings](https://account.bitvavo.com/user/api) to generate your API Key and Secret Key.
+
+<div class='note warning'>
+
+A word of caution: It is highly recommended to use the IP whitelist, and specify the lowest level of permissions you need. Using an API will bypass 2FA, keep your keys safe!
```
Comment (frenck) [inline source/_integrations/bitvavo.markdown]: > the lowest level of permissions you need

What would that lowest level of permissions be for this integration?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/bitvavo.markdown (hunk 1 above)
Comment (cyberjunky) [reply source/_integrations/bitvavo.markdown]: The warning is more generic, the integration only needs "Read Only" rights, which is the lowest level for Bitvavo.
Shall I change the text?

Comment (frenck) [reply source/_integrations/bitvavo.markdown]: I think this is better, as "you need" was a bit like: "I don't know what I need?" :)

Good change 👍

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed and not revived.

**Comment by cyberjunky** [comment]
Ah wait, I'm about to open the new PR for the Bitvavo integration

**Pull request closed without merging**